app/controllers/api/v1/dependencies_controller.rb

```diff
@@ -39,6 +45,7 @@ def cacheable!
   end
 
   def check_brownout
+    return if request.headers["x-dependency-api-allowed"]
     return if Patterns::JAVA_HTTP_USER_AGENT.match?(request.user_agent)
```

Comment:
we can delete this now, since we're only going based on the new header?

Reply:
we haven't deployed it yet, and I'd like to keep this as a fallback until after we are sure the fastly stuff works, but yes we can remove it after that's fully set up and tested and working.